pkg/gofr/datasource/sql/sql.go

```diff
@@ -33,13 +33,20 @@ type DBConfig struct {
 }
 
 func NewSQL(configs config.Config, logger datasource.Logger, metrics Metrics) *DB {
+	logger.Debugf("verifying database configurations from config file")
```

Comment:
since, we don't have any verification logic and are only reading the configs. We can replace `verifying` with `reading`